ote/requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy==1.19.5
-mmcv==1.3.9
```

Comment:
1.3.9 also works, seems like only the latest .10 is broken

Reply:
revert this change please